packages/code-du-travail-frontend/server/server.js

```diff
@@ -5,8 +5,8 @@ const { getKoaServer } = require("./koaServer");
  * this env variable is use to target developpement / staging deployement
  * in order to block indexing bot using a x-robot-header and an appropriate robots.txt
  */
-const IS_PRODUCTION_DEPLOYMENT =
-  process.env.IS_PRODUCTION_DEPLOYMENT === "true";
+const NEXT_PUBLIC_IS_PRODUCTION_DEPLOYMENT =
```

Comment:
On n'est pas obligé de mettre `NEXT_PUBLIC_IS_PRODUCTION_DEPLOYMENT` ici, c'est peut être même confusant. Je propose de garder l'ancien naming.

```suggestion
const IS_PRODUCTION_DEPLOYMENT =
```